hack/lib/release-images.sh

```diff
@@ -100,6 +102,11 @@ push_images() {
     docker push ${yurt_component_image}
 }
 
+kindload_images() {
+    build_images
+    kind load docker-image ${yurt_component_image} || { echo >&2 "kind not installed or error loading image: $(yurt_component_image)"; exit 1; }
```

Comment:
which node should we load image onto? maybe we need to specify the node by `--nodes xxx`